torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -134,38 +159,55 @@ c10::IValue getFunctionTuple(const Function& func) {
                       {"types", Tup(types)},
```

Comment:
It should be saved as part of this table, not separately.

Reply:
Can you clarify it a little bit, because I don't think I made any changes to this line?

Reply:
I mean your change saving it outside the main table. As as a new element in tuple. I am saying we should combine this with main table. I can help clarify if this is not clear.